e2e/explorer/workflow/active.spec.ts

```diff
@@ -1 +1,138 @@
+      name: namespace,
+    })
+    .click();
+
+  await expect(page, "the namespace is reflected in the url").toHaveURL(
+    `/${namespace}/explorer/tree`
+  );
+  await expect(
+    page.getByTestId("breadcrumb-namespace"),
+    "the namespace is reflected in the breadcrumbs"
+  ).toHaveText(namespace);
+
+  await page.getByTestId(`explorer-item-link-${workflow}`).click()
+  await expect(page.getByTestId('workflow-tabs-trg-activeRevision'), 'screen should have activeRevision tab').toBeVisible()
+});
+
+test.afterEach(async () => {
+  await deleteNamespace(namespace);
+  namespace = "";
+});
+
+const testSaveWorkflow = async (page: Page) =>{
+  const description = page.getByText(defaultDescription);
+  await description.click()
+  // type random text in that textarea which is for description
+  const testText = faker.random.alphaNumeric(9);
+  await page.type("textarea", testText);
+  
+  //now click on Save
+  const saveButton = page.getByTestId("workflow-editor-btn-save");
+  await saveButton.click();
+
+  // save button should be disabled/enabled while/after the api call
+  await expect(saveButton, 'save button should be disabled during the api call').toBeDisabled();
+  await expect(saveButton, 'save button should be enabled after the api call').toBeEnabled();
+  
+  // after saving is completed screen should have those new changed text before/after the page reload
+  await expect(page.getByText(testText), 'after saving, screen should have the updated text').toBeVisible();
+  await page.reload({waitUntil: 'load'});
+  await expect(page.getByText(testText), 'after reloading, screen should have the updated text').toBeVisible();
+
+  // check the text at the bottom left
+  await expect(page.getByTestId("workflow-txt-updated"), "text should be Updated a few seconds").toHaveText("Updated a few seconds");
+}
+
+const testMakeRevision = async (page: Page) =>{
+   const revisionTrigger = page.getByTestId("workflow-edit-trg-revision");
+  await revisionTrigger.click();
+  const makeRevisionButton = page.getByTestId("workflow-editor-btn-make-revision");
+  await makeRevisionButton.click();
+
+  // check the success toast action button
+  const toastAction = page.getByTestId("make-revision-toast-success-action");
+  await expect(toastAction, 'success toast should appear after make-revision button click').toBeVisible();
+
+  await toastAction.click();
+  await expect(toastAction, 'success toast should disappear after toast action click').toBeHidden();
+  await expect(page, "page url should have revision param").toHaveURL(/revision=/);
+  const revisionId = page.url().split('revision=')[1];
+  await expect(page.getByText(revisionId), "revisionId should be in the revision list").toBeVisible();
+  //go back to the workflow editor
+  await page.getByTestId('workflow-tabs-trg-activeRevision').click()
+}
+
+test("it is possible to save the workflow", async ({
+  page,
+}) => {
+  // click on the description so it can have input focus
```

Comment:
This comment is confusing, I would remove it or move it into the testSaveWorkflow method